src/ui/components/atoms/navbar/types/IProps.ts

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export interface IProps {
+  children?: React.ReactNode;
+  variant?: string;
```

Comment:
you have declared the variable variant here but you don't used no where in your component